test/system/doc_examples_axe_test.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "application_system_test_case"
+
+class IntegrationDocExamplesAxeTest < ApplicationSystemTestCase
+  # Skip components that make sense to test as part of a larger component.
+  NOT_STANDALONE = [:AutoCompleteItemPreview].freeze
+
+  # TEMP: Figure out how to load primer view helper & spinner in components
+  TEMP = [:OcticonComponentPreview, :ImagePreview, :ImageCropPreview].freeze
```

Comment:
todo: These currently raise errors because primer helpers and `spinner` undefined in preview template. Figure out how to include

[primer_octicon](https://github.com/primer/view_components/pull/587/files#diff-f7efe2ed4341d5d991490f7a390d0de659a29b2b328e1cf49cae7c95d7842c47R1)
[primer_image](https://github.com/primer/view_components/pull/587/files#diff-71b0785ad3fa46e83eea6a59695a2301a5868b282daa748914947c48407361faR2)
[spinner](https://github.com/primer/view_components/pull/587/files#diff-cd389216d8c8b5acc9fb38a7e83d01af8f628ecf122bc1443bf77b010c41cd41R2)